packages/wallet/src/redux/protocols/consensus-update/__tests__/reducer.test.ts

```diff
@@ -3,78 +3,189 @@ import { initialize, consensusUpdateReducer } from '../reducer';
 import * as states from '../states';
```

Comment:
It'd be nice to add `itSendsNoMessage` checks in any example that doesn't have an `itSendsMessage` check.